website/content/menu/index.md

```diff
@@ -10,14 +10,17 @@ headless = true
   - [Architecture]({{< relref "/documentation/MlosArchitecture" >}})
   - [Repo Organization]({{< relref "/documentation/RepoOrganization" >}})
 
-- Notebooks
+- [Notebooks]({{< relref "/notebooks" >}})
   - [Bayesian Optimization]({{< relref "/notebooks/BayesianOptimization" >}})
   - [Smart Cache Optimization]({{< relref "/notebooks/SmartCacheOptimization" >}})
 
-
 - API Documentation
   - [Python](/MLOS/python_api/)
 
+- [E2E Examples]({{< relref "/source/Examples" >}})
+  - [Smart Cache]({{< relref "/source/Examples/SmartCache" >}})
+  - [Smart Shared Channel]({{< relref "/source/Examples/SmartSharedChannel" >}})
```

Comment:
Is this one ready for the students? I haven't tested it yet.

Reply:
Hm... https://microsoft.github.io/MLOS/source/Examples/SmartSharedChannel/

Reply:
It's not as well documented, and isn't really meant as an educational tool, but they could use it.

Reply:
I think I would remove it for now if you don't mind.

Reply:
Done.